main.py

```diff
@@ -4,6 +4,10 @@
 def main():
     """Função principal que será rodada quando o script for passado para o interpretador."""
     # COLOQUE SEU CÓDIGO AQUI
+    x = float(input('Digite um número:'))
+    y = float(input('Digite outro número:'))
+    print(max(f'{x}',f'{y}'))
```

Comment:
Aqui temos um pequeno problema, apesar de passar nos testes automáticos o que acontece é que você compara o máximo entre duas *strings*, ou seja, vai ser a maior em ordem lexicográfica.
Ou seja, testando com `'5'` e `'200'` o 5 vai aparecer como se fosse o maior.

```suggestion
    print(f'{max(x, y)}')
```

Seria o jeito "certo" de se fazer isso.
Dentro da expressão vai o `max` que será calculado baseado nos valores numéricos.